test/testAuthCmd.js

```diff
@@ -151,4 +159,40 @@ describe('Integration test for auth', () => {
     assert.ok(out.data.indexOf('test-agent') > 0);
     assert.ok(out.data.indexOf(tokenLine) > 0);
   });
+
+  it('auth fires an error if GitHub user API has an issue', async () => {
+    const githubApi = nock('https://api.github.com')
+      .get('/user')
+      .reply(500, 'GitHub API has an issue');
+
+    const out = new TestStream();
+    out.isTTY = true;
+    const token = crypto.randomBytes(16).toString('hex');
+    const cmd = new AuthCommand()
+      .withDirectory(testRoot)
+      .withOpenBrowser(false)
+      .on('server-start', async (port) => {
+        const res = await fetchAPI.fetch(`http://127.0.0.1:${port}/`, {
+          method: 'POST',
+          json: {
+            token,
+          },
+        });
+        if (!res.ok) {
+          throw new Error(`Cannot request the server: ${await res.text()}`);
+        }
+        await res.buffer();
+        // required so that the server can stop properly
+        fetchAPI.disconnectAll();
+      });
+
+    try {
+      await cmd.run();
+      assert.fail('auth should fail');
+    } catch (e) {
+      assert.equal(e.message, '500 - "GitHub API has an issue"');
```

Comment:
this fails under windows: https://circleci.com/gh/adobe/helix-cli/13307?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

Reply:
Fixed! I had removed too much ;)